MRN-Code/coinstac
# Upgrade Router

Asana task: https://app.asana.com/0/45796090191289/58201701692527

### Goal

Upgrade React and React-ecosystem dependencies, especially the router. (It would be nice to use [redux-router](https://github.com/rackt/redux-router) to interact with routing as application state. Unfortunately, the lib causes breaking errors in Coinstac.)

### Testing

1. Pull the _feature/signup-and-login_ branch
2. Spin up [nodeapi](https://github.com/MRN-Code/nodeapi) with the Coinstac routes (`npm run startdev`)
3. Delete stored credentials:


    ```js
    localStorage.removeItem('COINSTAC_AUTH_RESPONSE');
    localStorage.removeItem('COINSTAC_USER');
    ```

4. Log in and observe routing works
5. Log out (you may see a “Stale timestamp” error from nodeapi)
